GRPCServiceA/pkg/user/endpoints.go

```diff
@@ -0,0 +1,42 @@
+package user
+
+import (
+	"context"
+
+	ent "github.com/coding-kiko/GoKit-Project-Bootcamp/GRPCServiceA/pkg/entities"
+	"github.com/go-kit/kit/endpoint"
+)
+
+type Endpoints struct {
+	CreateUser endpoint.Endpoint
+	GetUser    endpoint.Endpoint
+}
+
+func MakeEndpoints(s Service) Endpoints {
+	return Endpoints{
+		GetUser:    makeGetUserEndpoint(s),
+		CreateUser: makeCreateUserEndpoint(s),
+	}
+}
+
+func makeGetUserEndpoint(s Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(ent.GetUserReq)
```

Comment:
you are not validating the correct matching of the request? this could happen and generate an error.